src/jemalloc.c

```diff
@@ -1573,6 +1604,7 @@ malloc_init_hard_a0_locked() {
 	if (config_prof) {
 		prof_boot1();
 	}
+	thread_event_boot();
```

Comment:
Makes sense to boot later. I wonder if for the sake of being general we should always put it to the end?

Reply:
Didn't end up moving it further down. I'm actually thinking to kill the boot for thread_event, as it's only doing the all_events_off checking, which is only used for sanity checking.

BTW, what do you think use `te` as the short name for `thread_event`? I'm going to do some refactoring there and thinking about the shorten the names.